### DISCUSSION TITLE
是否支持后台运行？

### DISCUSSION THREAD

**Discussion by madjxatw** [zzzgydi/clash-verge#471] [General]
默认系统拖盘是开启的，从系统托盘退出后，clash core也停止。有没有类似CFW的light mode，即退出图形界面不影响后台运行，以可以节省内存。

**Comment by zzzgydi** [comment]
你是什么系统

**Reply by madjxatw** [reply]
Fedora 38 + i3wm

**Reply by zzzgydi** [reply]
linux的话，（理论上，我没测过）点叉关闭界面，就可以完全关闭图形界面且不影响后台运行了。

**Comment by madjxatw** [comment]
i3wm没有叉，关闭窗口后，系统托盘还在，是否表示UI还在运行？

**Reply by zzzgydi** [reply]
系统托盘还在（这个必须得在），说明程序后台也还在；ui还在不在，可能得查进程，有没有webview相关的进程还在跑。

**Reply by madjxatw** [reply]
那应该是支持后台运行的。打开dashboard时，clash-verge 会新建两个 webikit 相关的子进程，clash 本身也是 clash-verge 的子进程，所以 clash-verge 进程必须存在。
